Jenkinsfile

```diff
@@ -22,6 +22,7 @@ node('boardfarm') {
         sh 'echo "route add default gw 192.168.0.1" > /dev/ttyUSB0'
         sh 'sleep 10'
         sh 'echo "sed -i \'$ a nameserver 8.8.4.4\' /etc/resolv.conf" > /dev/ttyUSB0'
+        sh 'echo "sed -i \'/signature/d\' /etc/opkg.conf" > /dev/ttyUSB0'
```

Comment:
If you are using the v1.0.1 release image, then the signature check need not be by-passed?
The opkg test also has the purpose of verifying signature check, disabling it won't be a complete, true test